src/workflows/booking/cancel-booking.workflow.ts

```diff
@@ -0,0 +1,22 @@
+import { useBookingStore } from 'src/stores/booking.store';
+import { TheWorkflows } from '../the-workflows';
+
+TheWorkflows.on({
+  type: 'cancelBooking',
+  async cb(e) {
```

Comment:
```suggestion
  type: 'cancelBooking',
  loggable: 'operation',
  async cb(e) {
```
this will allow viewing operation history of the booking later.